www/nginx/src/opnsense/mvc/app/models/OPNsense/Nginx/Nginx.xml

```diff
@@ -1,6 +1,6 @@
 <model>
   <mount>//OPNsense/Nginx</mount>
-  <version>1.5.0</version>
+  <version>1.8.0</version>
```

Comment:
Unusual but why not...

Reply:
@fichtner In the nginx plugin, I always use the target plugin version to avoid duplicate version bumps for one release.